Classes/Private/Manager/Video/OCTVideoEngine.m

```diff
@@ -46,6 +46,11 @@ - (instancetype)init
 #if ! TARGET_OS_SIMULATOR
     _captureSession = [AVCaptureSession new];
     _captureSession.sessionPreset = AVCaptureSessionPresetMedium;
+
+    if ([_captureSession canSetSessionPreset:AVCaptureSessionPreset640x480])
+    {
+        _captureSession.sessionPreset = AVCaptureSessionPreset640x480;
```

Comment:
Hm, GitHub has confusing review interface, my comment was removed. Anyway, let's try 640x480 for now, I'll play with other options later.